editor/src/components/canvas/controls/classname-select.tsx

```diff
@@ -428,25 +460,33 @@ export const ClassNameSelect: React.FunctionComponent = betterReactMemo('ClassNa
         '&:focus-within': { boxShadow: `0px 0px 0px 1px ${theme.primary.value}` },
       }}
     >
-      <WindowedSelect
-        filterOption={filterOption}
-        options={filteredOptions}
-        onChange={onChange}
-        onInputChange={setInput}
-        value={selectedValues}
-        isMulti={true}
-        isDisabled={!isMenuEnabled}
-        closeMenuOnSelect={false}
-        styles={colourStyles}
-        components={{
-          DropdownIndicator,
-          ClearIndicator,
-          IndicatorSeparator,
-          NoOptionsMessage,
-          MultiValueContainer,
-          ValueContainer,
+      <ClassNameMenuContext.Provider
+        value={{
+          focused: focusedOption,
```

Comment:
I'm not sure how much the context menu items rerender on hover change to begin with. but anyone listening to this context will rerender whenever ClassNameSelect rerenders: in the new-ish React APIs, like useStatea and useContext the value is compared using `===` strict reference equality for contexts. 
The simplest workaround for now is that the type of value should not be `{focused: TailWindOption}`, instead just `TailWindOption`

Reply:
Testing now with a component that consumes the context

Reply:
Just verified and it doesn't cause unnecessary re-renders as is. I believe because of our patching of the `useContextSelector`.